tests/centraldashboard-base_test.go

```diff
@@ -76,8 +76,11 @@ spec:
           value: $(userid-header)
         - name: USERID_PREFIX
           value: $(userid-prefix)
+<<<<<<< HEAD
         - name: PROFILES_KFAM_SERVICE_HOST
           value: profiles-kfam.kubeflow
+=======
```

Comment:
@jlewi This seems to be a relic of a merge conflict that wasn't resolved and committed?